<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayFair Cypher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>700</width>
      <height>330</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Data</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Text / File Path</string>
     </property>
     <widget class="QPlainTextEdit" name="inputTextField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>681</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="executeButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>280</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Execute</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Password</string>
     </property>
     <widget class="QLineEdit" name="inputPasswordField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="types">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="inputTypes">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input type</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>700</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>681</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Encrypted / Decrypted Text</string>
     </property>
     <widget class="QPlainTextEdit" name="outputTextField">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>681</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="saveToFileField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveToFileButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>210</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>681</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File path (to save the results)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="inputDataFormatInfo"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInput_data_format">
   <property name="text">
    <string>Input data format</string>
   </property>
  </action>
  <action name="inputDataFormatInfo">
   <property name="text">
    <string>Input Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
